xblock/mixins.py

```diff
@@ -538,3 +538,50 @@ def index_dictionary(self):
         default implementation is an empty dict
         """
         return {}
+
+
+class ViewsMixin(object):
+    """
+    This mixin provides decorators that can be used on xBlock view methods.
+    """
+    @classmethod
+    def supports(cls, functionality):
```

Comment:
Nit: should this be `*functionalities`, so that you can do something like `@supports('multi_device', 'screen_reader', 'lynx')`?